coc/src/vendor/machineId.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
We should add a section attributing this file to the machine-id node module.

I will ping the channel to get the right formulation.